frontend/src/App.tsx

```diff
@@ -4,21 +4,29 @@ import {BrowserRouter, Route, Routes} from "react-router-dom";
 import SignUp from "./SignUp";
 import NavigationBar from "./NavigationBar";
 import {Typography} from "@mui/material";
+import LoginPage from './LoginPage';
+import OwnerUse from './OwnerUse';
+import AddLocation from './AddLocation';
 
+import LocationUse from './LocationUse';
 
-function App() {
 
+function App() {
 
+    const {login, addOwner} = OwnerUse()
+    const {addLocation} = LocationUse()
 
-  return (
+    return (
 
-      <BrowserRouter >
-              <header className={"header"}>
-                  <NavigationBar />
-              </header>
-                  <div>
-                      <Routes>
-                      <Route path = {"owners/register/"} element= {<SignUp/>}/>
+        <BrowserRouter>
+            <header className={"header"}>
+                <NavigationBar/>
+            </header>
+            <div>
+                <Routes>
+                    <Route path={"api/owners/register/"} element={<SignUp addOwner={addOwner}/>}/>
+                    <Route path={"api/owners/login"} element={<LoginPage login={login}/>}/>
+                    <Route path={"api/locations/newlocation"} element={<AddLocation addLocation={addLocation}/>} />
```

Comment:
Das `api/` davor sollte nur im BE benutzt werden, da es halt die API ist. Dieser Prefix steht ja auch dafür damit wir zwischen Frontendroute und API ROute unterscheiden können

Reply:
Füg hier gerne auch noch eine Standardroute hinzu (also für `/`)